Script/AtomicEditor/ui/frames/menus/HierarchyFrameMenu.ts

```diff
@@ -110,7 +115,8 @@ class HierarchyFrameMenus extends Atomic.ScriptObject {
                 node.scene.sendEvent("SceneEditNodeCreated", { node: newnode });
             }
 
-            return true;
+            // Let plugins handle context
+            return ServiceLocator.uiServices.hierarchyContextItemClicked(target, refid);
```

Comment:
Great suggestion, that'll simplify use a lot. And good catch on the return. I'll make those changes tomorrow, I'm out of office today.